lib/net/pool.js

```diff
@@ -1291,7 +1402,7 @@ class Pool extends EventEmitter {
     }
 
     // We want compact blocks!
-    if (this.options.compact)
+    if (this.options.compact && !this.options.neutrino)
```

Comment:
huh that's funny - does SPV send `sendcompact` packets even though we dont get blocks at all? Whoops